README.md

```diff
@@ -65,9 +694,9 @@ We have the tool implemented in 4 Technologies:
 - Node(Typescript): https://github.com/pin-people/event_people_node
 - Go: https://github.com/pin-people/event_people_go
 
-But if you want to use it on your project and the Technology is not supported, don't worry, below is the class Diagram so you can implement it yourself:
+But if you want to use it on your project and the Technology is not supported yet, don't worry, you can use the guidelines in this repository to implement it!
 
-![event_people class' diagram](diagram.png "EventPeople Class' Diagram")
+---
 
 ## License
```

Comment:
Is there any reason to choose the LGPL 3.0 License?

Reply:
The main reason is to keep it Open Source, as follows:

> 2. Conveying Modified Versions.
> 
>  If you modify a copy of the Library, and, in your modifications, a
facility refers to a function or data to be supplied by an Application
that uses the facility (other than as an argument passed when the
facility is invoked), then you may convey a copy of the modified
version:
>
 >  a) under this License, provided that you make a good faith effort to
   ensure that, in the event an Application does not supply the
   function or data, the facility still operates, and performs
   whatever part of its purpose remains meaningful